src/reduxify-navigator.js

```diff
@@ -43,7 +43,9 @@ function reduxifyNavigator<State: NavigationState, Props: RequiredProps<State>>(
     }
 
     componentDidUpdate() {
-      didUpdateCallback();
+      setTimeout(() => {
+        didUpdateCallback();
```

Comment:
I don’t think there is a reason to construct a new function here. Try just passing `didUpdateCallback` in directly and see if it typechecks with `flow`.